public/app/types/user.ts

```diff
@@ -106,13 +107,14 @@ export interface UserAdminError {
   body: string;
 }
 
+export type UserFilter = Record<string, string | boolean | SelectableValue[]>;
```

Comment:
This doesn't feel right. Can't we just use boolean or SelectableValue?

Reply:
Selectable value will not work if a filter has multiple values, e.g. for licensed role we want to allow filtering by multiple roles (the filter is a multi select), so the filter can be for example: `licensedRole=viewer&licensedRole=admin_editor`. Using `Record<string, string>` doesn't work here. We could store the filter in this case as `{licensedRole: 'viewer,admin_editor'}` but that adds extra processing logic and by keeping it a selectable value we can pass it into select components without any processing.

Reply:
Here's [an example](https://github.com/grafana/grafana-enterprise/pull/2010/files#diff-8717404412529f3c36b9c9a64d0ef41ad8649c0db103a00016a80b75aa14f3f5R12) of a multi select filter.

Reply:
Can we use SelectableValue[] then?

Reply:
That's not enough because for multi select filters it'll be e.g. `[{Admin: 'admin'}]`, which doesn't have the name of the filter itself.